.gitignore

```diff
@@ -2,8 +2,12 @@
 **/.DS_Store
 **/xcuserdata/
 BowPlayground
-/.build
+
+## SPM
+.build
+Package.resolved
```

Comment:
Apparently, `Package.resolved` should not be ignored to ensure reproducibility of builds.

Reply:
It is a different case, in this case, this `Package.resolved` is generated after build nef library to check it; it is not the dependencies for the project - we are using Xcode SPM.